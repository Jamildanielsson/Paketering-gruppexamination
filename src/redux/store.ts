import { configureStore } from '@reduxjs/toolkit';
import latestMovieReducer from './latestMovieSlice';

const store = configureStore({
  reducer: {
    lastestMovie: latestMovieReducer,
  },
});

// FÃ¶r att redux ska funka med ts...
export type RootState = ReturnType<typeof store.getState>;
export type AppDispatch = typeof store.dispatch;

export default store;
