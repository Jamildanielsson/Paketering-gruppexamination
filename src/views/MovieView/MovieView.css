.movie-view__add-button {
  -webkit-appearance: button;
     -moz-appearance: button;
          appearance: button;
  backface-visibility: hidden;
  background-color: rgb(177, 31, 36);
  border-radius: 6px;
  border-width: 0;
  box-shadow: rgba(50, 50, 93, 0.1) 0 0 0 1px inset, rgba(50, 50, 93, 0.1) 0 2px 5px 0, rgba(0, 0, 0, 0.07) 0 1px 1px 0;
  box-sizing: border-box;
  color: #fff;
  cursor: pointer;
  font-family: -apple-system, system-ui, "Segoe UI", Roboto, "Helvetica Neue", Ubuntu, sans-serif;
  font-size: 100%;
  height: 44px;
  line-height: 1.15;
  margin: 12px 0 0;
  outline: none;
  overflow: hidden;
  padding: 0 25px;
  position: relative;
  text-align: center;
  text-transform: none;
  transform: translateZ(0);
  transition: all 0.2s, box-shadow 0.08s ease-in;
  -moz-user-select: none;
       user-select: none;
  -webkit-user-select: none;
  touch-action: manipulation;
  max-width: 50%;
  margin: 0.2rem;
}
@media screen and (max-width: 900px) {
  .movie-view__add-button {
    margin-left: 0rem;
  }
}
.movie-view__add-button:disabled {
  cursor: default;
}
.movie-view__add-button:focus {
  box-shadow: rgba(50, 50, 93, 0.1) 0 0 0 1px inset, rgba(50, 50, 93, 0.2) 0 6px 15px 0, rgba(0, 0, 0, 0.1) 0 2px 2px 0, rgba(50, 151, 211, 0.3) 0 0 0 4px;
}
.movie-view__delete-button {
  margin: 0.2rem;
  -webkit-appearance: button;
     -moz-appearance: button;
          appearance: button;
  backface-visibility: hidden;
  background-color: rgb(228, 30, 38);
  border-radius: 6px;
  border-width: 0;
  box-shadow: rgba(50, 50, 93, 0.1) 0 0 0 1px inset, rgba(50, 50, 93, 0.1) 0 2px 5px 0, rgba(0, 0, 0, 0.07) 0 1px 1px 0;
  box-sizing: border-box;
  color: #fff;
  cursor: pointer;
  font-family: -apple-system, system-ui, "Segoe UI", Roboto, "Helvetica Neue", Ubuntu, sans-serif;
  font-size: 100%;
  height: 44px;
  line-height: 1.15;
  margin: 12px 0 0;
  outline: none;
  overflow: hidden;
  padding: 0 25px;
  position: relative;
  text-align: center;
  text-transform: none;
  transform: translateZ(0);
  transition: all 0.2s, box-shadow 0.08s ease-in;
  -moz-user-select: none;
       user-select: none;
  -webkit-user-select: none;
  touch-action: manipulation;
  max-width: 50%;
}
@media screen and (max-width: 900px) {
  .movie-view__delete-button {
    margin-left: 0rem;
  }
}
.movie-view__delete-button:disabled {
  cursor: default;
}
.movie-view__delete-button:focus {
  box-shadow: rgba(50, 50, 93, 0.1) 0 0 0 1px inset, rgba(50, 50, 93, 0.2) 0 6px 15px 0, rgba(0, 0, 0, 0.1) 0 2px 2px 0, rgba(50, 151, 211, 0.3) 0 0 0 4px;
}
.movie-view__card {
  padding: 20px;
  margin: auto;
  max-width: 80rem;
  box-shadow: 0px 4px 10px rgba(134, 134, 134, 0.5);
  font-family: Verdana, Geneva, Tahoma, sans-serif;
  margin-top: 20px;
}
.movie-view__card__container {
  display: grid;
  grid-template-columns: repeat(3, 1fr);
}
@media screen and (max-width: 900px) {
  .movie-view__card__container {
    display: flex;
    flex-direction: column;
    align-items: center;
    text-align: left;
  }
}
.movie-view__card__container__year p {
  margin-left: 10rem;
  padding: 0;
}
.movie-view__card__container__thumbnail {
  grid-column: 1/2;
}
@media screen and (max-width: 900px) {
  .movie-view__card__container__thumbnail {
    display: flex;
    flex-direction: column;
    max-width: 100%;
  }
}
.movie-view__card__container__thumbnailIMG {
  max-width: 30rem;
}
.movie-view__card__container__title {
  font-size: 3rem;
}
@media screen and (max-width: 900px) {
  .movie-view__card__container__title {
    font-size: 1.5rem;
    margin-bottom: 1rem;
  }
}
.movie-view__card__container__main-info {
  margin: 1rem;
  font-size: 1.3rem;
}
@media screen and (max-width: 900px) {
  .movie-view__card__container__main-info {
    font-size: 1rem;
    margin-left: 0;
  }
}
.movie-view__card__container__info {
  grid-column: 2/4;
  display: flex;
  flex-direction: column;
  justify-content: space-between;
  margin-left: 2rem;
}
@media screen and (max-width: 900px) {
  .movie-view__card__container__info {
    margin-left: 0;
  }
}
.movie-view__card__container__info p {
  font-size: 1rem;
  margin-bottom: 1.3rem;
  margin: 1rem;
  margin-left: 0;
}/*# sourceMappingURL=MovieView.css.map */