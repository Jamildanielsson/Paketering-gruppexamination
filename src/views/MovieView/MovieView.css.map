{"version":3,"sources":["MovieView.scss","../../sass/_colors.scss","MovieView.css"],"names":[],"mappings":"AAIE;EACA,0BAAA;KAAA,uBAAA;UAAA,kBAAA;EACA,2BAAA;EACA,kCCDY;EDEZ,kBAAA;EACA,eAAA;EACA,qHAAA;EACA,sBAAA;EACA,WAAA;EACA,eAAA;EACA,+FAAA;EACA,eAAA;EACA,YAAA;EACA,iBAAA;EACA,gBAAA;EACA,aAAA;EACA,gBAAA;EACA,eAAA;EACA,kBAAA;EACA,kBAAA;EACA,oBAAA;EACA,wBAAA;EACA,8CAAA;EACA,sBAAA;OAAA,iBAAA;EACA,yBAAA;EACA,0BAAA;EACA,cAAA;EACA,cAAA;AEHF;AFIE;EA5BA;IA6BE,iBAAA;EEDF;AACF;AFIA;EACE,eAAA;AEFF;AFKA;EACE,wJAAA;AEHF;AFKA;EACE,cAAA;EACA,0BAAA;KAAA,uBAAA;UAAA,kBAAA;EACA,2BAAA;EACA,kCC3CQ;ED4CR,kBAAA;EACA,eAAA;EACA,qHAAA;EACA,sBAAA;EACA,WAAA;EACA,eAAA;EACA,+FAAA;EACA,eAAA;EACA,YAAA;EACA,iBAAA;EACA,gBAAA;EACA,aAAA;EACA,gBAAA;EACA,eAAA;EACA,kBAAA;EACA,kBAAA;EACA,oBAAA;EACA,wBAAA;EACA,8CAAA;EACA,sBAAA;OAAA,iBAAA;EACA,yBAAA;EACA,0BAAA;EACC,cAAA;AEHH;AFIG;EA5BH;IA6BI,iBAAA;EEDF;AACF;AFIA;EACE,eAAA;AEFF;AFKA;EACE,wJAAA;AEHF;AFKE;EACE,aAAA;EACA,YAAA;EACA,gBAAA;EACA,iDAAA;EACA,gDAAA;EACA,gBAAA;AEHJ;AFII;EACE,aAAA;EACA,qCAAA;AEFN;AFGM;EAHF;IAII,aAAA;IACA,sBAAA;IACA,mBAAA;IACA,gBAAA;EEAN;AACF;AFCI;EACE,kBAAA;EACA,UAAA;AECN;AFCI;EACE,gBAAA;AECN;AFAM;EAFF;IAGI,aAAA;IACA,sBAAA;IACA,eAAA;EEGN;AACF;AFDI;EACC,gBAAA;AEGL;AFDM;EACE,eAAA;AEGR;AFFQ;EAFF;IAGI,iBAAA;IACA,mBAAA;EEKR;AACF;AFHM;EACE,YAAA;EACA,iBAAA;AEKR;AFJQ;EAHF;IAII,eAAA;IACA,cAAA;EEOR;AACF;AFLM;EACE,gBAAA;EACA,aAAA;EACA,sBAAA;EACA,8BAAA;EACA,iBAAA;AEOR;AFNQ;EANF;IAOE,cAAA;EESN;AACF;AFPM;EACE,eAAA;EACA,qBAAA;EACA,YAAA;EACA,cAAA;AESR","file":"MovieView.css"}