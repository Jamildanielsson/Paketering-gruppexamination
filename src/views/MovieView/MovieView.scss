@import '../../sass/colors';
$medium: 850px;

.movie-view {
  &__add-and-delete-button {
    background-color: $logoDarkRed;
    border-radius: 6px;
    border-width: 0;
    box-shadow: rgba(50, 50, 93, 0.1) 0 0 0 1px inset,
      rgba(50, 50, 93, 0.1) 0 2px 5px 0, rgba(0, 0, 0, 0.07) 0 1px 1px 0;
    color: $white;
    font-size: 100%;
    height: 3rem;
    margin: 12px 0 0;
    padding: 0 25px;
    user-select: none;
    -webkit-user-select: none;
    touch-action: manipulation;
    width: 30%;
    margin: 0.2rem;

    &:hover {
      cursor: pointer;
    }

    @media screen and (max-width: $medium) {
      margin-left: 0rem;
      width: 100%;
    }
  }

  &__card {
    padding: 20px;
    margin: auto;
    max-width: 80rem;
    box-shadow: 0px 4px 10px rgba(134, 134, 134, 0.5);
    font-family: Verdana, Geneva, Tahoma, sans-serif;
    margin-top: 20px;

    &__container {
      display: grid;
      grid-template-columns: repeat(3, 1fr);
      @media screen and (max-width: $medium) {
        display: flex;
        flex-direction: column;
        align-items: center;
        text-align: left;
      }
      &__year p {
        margin-left: 10rem;
        padding: 0;
      }
      &__thumbnail {
        grid-column: 1/2;
        @media screen and (max-width: $medium) {
          display: flex;
          flex-direction: column;
          max-width: 100%;
        }
      }
      &__thumbnailIMG {
        max-width: 52vh;
      }
      &__title {
        font-size: 3rem;
        @media screen and (max-width: $medium) {
          font-size: 1.5rem;
          margin-bottom: 1rem;
        }
      }
      &__main-info {
        margin: 1rem;
        font-size: 1.3rem;
        @media screen and (max-width: $medium) {
          font-size: 1rem;
          margin-left: 0;
        }
      }
      &__info {
        grid-column: 2/4;
        display: flex;
        flex-direction: column;
        justify-content: space-between;
        margin-left: 2rem;
        @media screen and (max-width: $medium) {
          margin-left: 0;
        }
      }
      &__info p {
        font-size: 1rem;
        margin-bottom: 1.3rem;
        margin: 1rem;
        margin-left: 0;
      }
    }
  }
}
