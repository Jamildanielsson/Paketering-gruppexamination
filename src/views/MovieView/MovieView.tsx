import Header from '../../components/Header/Header';
import './MovieView.scss';
import { RootState } from '../../redux/store';
import { useSelector } from 'react-redux';
import { useEffect, useState } from 'react';
import { findMovie } from '../../utils/findMovie';

function MovieView() {
  const isFavorited: boolean = false;
  const [movie, setMovie] = useState<MovieData | undefined>(undefined);

  const latestClickedMovie: string = useSelector(
    (state: RootState) => state.lastestMovie.latestMovie
  );

  useEffect(() => {
    setMovie(findMovie(latestClickedMovie));
  }, [latestClickedMovie]);

  function addToFavorites() {
    // TODO: Add the movie to favorites...
  }

  function removeFromFavorites() {
    // TODO: Remove the movie from favorites...
  }

  return (
    <div className='movie-view'>
      <Header />

      {movie ? (
        <section className='movie-view__card'>
          <div className='movie-view__card__container'>
            <div className='movie-view__card__container__thumbnail'>
              <img
                className='movie-view__card__container__thumbnailIMG'
                src={movie.thumbnail}
                alt={movie.title}
              />
            </div>
            <div className='movie-view__card__container__info'>
              <div className='movie-view__card__container__main-info'>
                <h1 className='movie-view__card__container__title'>
                  {movie.title}
                </h1>
                <p className='movie-view__card__container__year'>
                  {movie.year} | {movie.genre} |  {movie.rating}
                </p>
                <h4 className='movie-view__card__container__actors'>
                  {movie.actors.join(', ')}
                </h4>
              </div>
              <div>
                <p className='movie-view__synopsis'>{movie.synopsis}</p>

                {isFavorited ? (
                  <button
                    className='movie-view__delete-button'
                    onClick={() => removeFromFavorites()}
                  >
                    Remove from Favorites
                  </button>
                ) : (
                  <button
                    className='movie-view__add-button'
                    onClick={() => addToFavorites()}
                  >
                    Add to Favorites
                  </button>
                )}
              </div>
            </div>
          </div>
        </section>
      ) : (
        '' // Todo: Fixa n√•gon snygg placeholder...

      )}




    </div >
  );
}

export default MovieView;
