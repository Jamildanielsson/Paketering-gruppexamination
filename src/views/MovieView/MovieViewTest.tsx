
import { describe, it, expect } from 'vitest';
import { render, screen } from '@testing-library/react'
import MovieView from './MovieView';
import user from "@testing-library/user-event";


// test add and remove bookmarks
// test movie is showing

describe('Check search', () => {
  it("Should display error message if search bar is empty", async () => {
    render(<MovieView />);

    const searchBtn = screen.getByRole("button");
    await user.click(searchBtn);
   
    expect(await screen.findByText("Sorry pal, we couldn't see any Bookmarks ")).toBeInTheDocument();
  });

  it("Should show a title 'The Shawshank Redemption' ");
  render(<MovieView />);

  // it("Should display error message if search bar contains word with no definition", async () => {
  //   render(<MovieView />);

  //   const input = screen.getByPlaceholderText('Search...');
  //   const searchBtn = screen.getByRole("button");
  //   await user.type(input, 'asd');
  //   await user.click(searchBtn);
   
  //   expect(await screen.findByText("Sorry pal, we couldn't find definitions for the word you were looking for.")).toBeInTheDocument();
  // });

});