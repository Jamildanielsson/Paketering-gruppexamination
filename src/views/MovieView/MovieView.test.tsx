import { render, screen } from '@testing-library/react';
import userEvent from '@testing-library/user-event';
import { describe, expect, it } from 'vitest';
import { BrowserRouter } from 'react-router-dom';
import MovieView from './MovieView';
import { Provider } from 'react-redux';
import store from '../../redux/store';


describe('Movie view', () => {
    it('Should render the movie view', async () => {
        render(
            <Provider store={store}>
                <BrowserRouter>
                <MovieView />
                </BrowserRouter>
                </Provider>
        )
        

    })
})