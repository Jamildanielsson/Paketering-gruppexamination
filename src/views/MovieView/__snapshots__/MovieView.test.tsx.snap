// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Movie view > Should render app 1`] = `
<div>
  <header
    className="header"
  >
    <img
      alt="Fakeflix logo"
      className="header__logo"
      src="/Paketering-gruppexamination/src/assets/images/fakeflix.png"
    />
    <section
      className="hamburger-menu "
      onClick={[Function]}
    >
      <span
        className="hamburger-menu__line"
      />
      <span
        className="hamburger-menu__line"
      />
      <span
        className="hamburger-menu__line"
      />
    </section>
    <nav
      className="navigation"
    />
  </header>
  <div
    className="Searchbar"
  >
    <div
      className="input-wrapper"
    >
      <svg
        fill="currentColor"
        height="1em"
        id="SearchIcon"
        stroke="currentColor"
        strokeWidth="0"
        style={
          {
            "color": undefined,
          }
        }
        viewBox="0 0 512 512"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M505 442.7L405.3 343c-4.5-4.5-10.6-7-17-7H372c27.6-35.3 44-79.7 44-128C416 93.1 322.9 0 208 0S0 93.1 0 208s93.1 208 208 208c48.3 0 92.7-16.4 128-44v16.3c0 6.4 2.5 12.5 7 17l99.7 99.7c9.4 9.4 24.6 9.4 33.9 0l28.3-28.3c9.4-9.4 9.4-24.6.1-34zM208 336c-70.7 0-128-57.2-128-128 0-70.7 57.2-128 128-128 70.7 0 128 57.2 128 128 0 70.7-57.2 128-128 128z"
        />
      </svg>
      <input
        onChange={[Function]}
        placeholder="Search movie on title?"
        value=""
      />
    </div>
    <div
      className="searchResults"
    >
      <ul />
    </div>
  </div>
  <h1
    className="home-heading"
  >
    Trending:
  </h1>
  <div
    className="megabody-t"
  >
    <div
      className="container-t"
    >
      <div
        className="slider-wrapper-t"
      >
        <button
          className="slide-button-t material-symbols-rounded"
          id="prev-slide"
        >
          chevron_left
        </button>
        <div
          className="image-list-t"
        >
          <img
            alt="Movie 0"
            className="image-item-t"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BMWMwMGQzZTItY2JlNC00OWZiLWIyMDctNDk2ZDQ2YjRjMWQ0XkEyXkFqcGdeQXVyNzkwMjQ5NzM@._V1_QL75_UY562_CR7,0,380,562_.jpg"
          />
          <img
            alt="Movie 1"
            className="image-item-t"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BNzA5ZDNlZWMtM2NhNS00NDJjLTk4NDItYTRmY2EwMWZlMTY3XkEyXkFqcGdeQXVyNzkwMjQ5NzM@._V1_SY1000_CR0,0,675,1000_AL_.jpg"
          />
          <img
            alt="Movie 2"
            className="image-item-t"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/I/71V4mbT4n9L._AC_SL1130_.jpg"
          />
          <img
            alt="Movie 3"
            className="image-item-t"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BYmU1NDRjNDgtMzhiMi00NjZmLTg5NGItZDNiZjU5NTU4OTE0XkEyXkFqcGdeQXVyNzkwMjQ5NzM@._V1_SY1000_CR0,0,675,1000_AL_.jpg"
          />
          <img
            alt="Movie 4"
            className="image-item-t"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BY2IzZGY2YmEtYzljNS00NTM5LTgwMzUtMzM1NjQ4NGI0OTk0XkEyXkFqcGdeQXVyNDYyMDk5MTU@._V1_QL75_UX380_CR0,5,380,562_.jpg"
          />
          <img
            alt="Movie 5"
            className="image-item-t"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BMGU2NzRmZjUtOGUxYS00ZjdjLWEwZWItY2NlM2JhNjkxNTFmXkEyXkFqcGdeQXVyNjU0OTQ0OTY@._V1_QL75_UX380_CR0,1,380,562_.jpg"
          />
          <img
            alt="Movie 6"
            className="image-item-t"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BNTQwNDM1YzItNDAxZC00NWY2LTk0M2UtNDIwNWI5OGUyNWUxXkEyXkFqcGdeQXVyNzkwMjQ5NzM@._V1_QL75_UX380_CR0,1,380,562_.jpg"
          />
        </div>
        <button
          className="slide-button-t material-symbols-rounded"
          id="next-slide"
        >
          chevron_right
        </button>
      </div>
      <div
        className="slider-scrollbar-t"
      >
        <div
          className="scrollbar-track-t"
        >
          <div
            className="scrollbar-thumb-t"
          />
        </div>
      </div>
    </div>
  </div>
  <h1
    className="home-heading"
  >
    Recommended:
  </h1>
  <div
    className="megabody"
  >
    <div
      className="container"
    >
      <div
        className="slider-wrapper"
      >
        <button
          className="slide-button material-symbols-rounded"
          id="prev-slide"
        >
          chevron_left
        </button>
        <div
          className="image-list"
        >
          <img
            alt="Movie 0"
            className="image-item"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BNDE3ODcxYzMtY2YzZC00NmNlLWJiNDMtZDViZWM2MzIxZDYwXkEyXkFqcGdeQXVyNjAwNDUxODI@._V1_QL75_UX380_CR0,4,380,562_.jpg"
          />
          <img
            alt="Movie 1"
            className="image-item"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BM2MyNjYxNmUtYTAwNi00MTYxLWJmNWYtYzZlODY3ZTk3OTFlXkEyXkFqcGdeQXVyNzkwMjQ5NzM@._V1_QL75_UY562_CR8,0,380,562_.jpg"
          />
          <img
            alt="Movie 2"
            className="image-item"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BMTMxNTMwODM0NF5BMl5BanBnXkFtZTcwODAyMTk2Mw@@._V1_SY1000_CR0,0,675,1000_AL_.jpg"
          />
          <img
            alt="Movie 3"
            className="image-item"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BMWU4N2FjNzYtNTVkNC00NzQ0LTg0MjAtYTJlMjFhNGUxZDFmXkEyXkFqcGdeQXVyNjc1NTYyMjg@._V1_SY1000_CR0,0,675,1000_AL_.jpg"
          />
          <img
            alt="Movie 4"
            className="image-item"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BNDE4OTMxMTctNmRhYy00NWE2LTg3YzItYTk3M2UwOTU5Njg4XkEyXkFqcGdeQXVyNjU0OTQ0OTY@._V1_SY1000_CR0,0,675,1000_AL_.jpg"
          />
          <img
            alt="Movie 5"
            className="image-item"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BNGNhMDIzZTUtNTBlZi00MTRlLWFjM2ItYzViMjE3YzI5MjljXkEyXkFqcGdeQXVyNzkwMjQ5NzM@._V1_SY1000_CR0,0,675,1000_AL_.jpg"
          />
          <img
            alt="Movie 6"
            className="image-item"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BOTQ5NDI3MTI4MF5BMl5BanBnXkFtZTgwNDQ4ODE5MDE@._V1_SY1000_CR0,0,675,1000_AL_.jpg"
          />
          <img
            alt="Movie 7"
            className="image-item"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BNWIwODRlZTUtY2U3ZS00Yzg1LWJhNzYtMmZiYmEyNmU1NjMzXkEyXkFqcGdeQXVyMTQxNzMzNDI@._V1_SY1000_CR0,0,675,1000_AL_.jpg"
          />
          <img
            alt="Movie 8"
            className="image-item"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BMjAxMzY3NjcxNF5BMl5BanBnXkFtZTcwNTI5OTM0Mw@@._V1_SY1000_CR0,0,675,1000_AL_.jpg"
          />
          <img
            alt="Movie 9"
            className="image-item"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BN2EyZjM3NzUtNWUzMi00MTgxLWI0NTctMzY4M2VlOTdjZWRiXkEyXkFqcGdeQXVyNDUzOTQ5MjY@._V1_SY1000_CR0,0,675,1000_AL_.jpg"
          />
          <img
            alt="Movie 10"
            className="image-item"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BNzQzOTk3OTAtNDQ0Zi00ZTVkLWI0MTEtMDllZjNkYzNjNTc4L2ltYWdlXkEyXkFqcGdeQXVyNjU0OTQ0OTY@._V1_SY1000_CR0,0,675,1000_AL_.jpg"
          />
          <img
            alt="Movie 11"
            className="image-item"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BY2NkZjEzMDgtN2RjYy00YzM1LWI4ZmQtMjIwYjFjNmI3ZGEwXkEyXkFqcGdeQXVyNzkwMjQ5NzM@._V1_SY1000_CR0,0,675,1000_AL_.jpg"
          />
          <img
            alt="Movie 12"
            className="image-item"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BZjA0OWVhOTAtYWQxNi00YzNhLWI4ZjYtNjFjZTEyYjJlNDVlL2ltYWdlL2ltYWdlXkEyXkFqcGdeQXVyMTQxNzMzNDI@._V1_SY1000_CR0,0,675,1000_AL_.jpg"
          />
          <img
            alt="Movie 13"
            className="image-item"
            onClick={[Function]}
            src="https://upload.wikimedia.org/wikipedia/commons/thumb/b/ba/Seven_Samurai_poster.jpg/1024px-Seven_Samurai_poster.jpg"
          />
          <img
            alt="Movie 14"
            className="image-item"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BNjNhZTk0ZmEtNjJhMi00YzFlLWE1MmEtYzM1M2ZmMGMwMTU4XkEyXkFqcGdeQXVyNjU0OTQ0OTY@._V1_SY1000_CR0,0,675,1000_AL_.jpg"
          />
          <img
            alt="Movie 15"
            className="image-item"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BZGMxZTdjZmYtMmE2Ni00ZTdkLWI5NTgtNjlmMjBiNzU2MmI5XkEyXkFqcGdeQXVyNjU0OTQ0OTY@._V1_QL75_UX380_CR0,14,380,562_.jpg"
          />
          <img
            alt="Movie 16"
            className="image-item"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BNTU2ODkyY2MtMjU1NC00NjE1LWEzYjgtMWQ3MzRhMTE0NDc0XkEyXkFqcGdeQXVyMjM4MzQ4OTQ@._V1_QL75_UY562_CR1,0,380,562_.jpg"
          />
          <img
            alt="Movie 17"
            className="image-item"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BMGU5OWEwZDItNmNkMC00NzZmLTk1YTctNzVhZTJjM2NlZTVmXkEyXkFqcGdeQXVyMTMxODk2OTU@._V1_SY1000_CR0,0,675,1000_AL_.jpg"
          />
          <img
            alt="Movie 18"
            className="image-item"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BODQ3NDExOGYtMzI3Mi00NWRlLTkwNjAtNjc4MDgzZGJiZTA1XkEyXkFqcGdeQXVyMjUzOTY1NTc@._V1_QL75_UX380_CR0,3,380,562_.jpg"
          />
          <img
            alt="Movie 19"
            className="image-item"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BMTI1MTY2OTIxNV5BMl5BanBnXkFtZTYwNjQ4NjY3._V1_QL75_UY562_CR0,0,380,562_.jpg"
          />
          <img
            alt="Movie 20"
            className="image-item"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BNGUxYWM3M2MtMGM3Mi00ZmRiLWE0NGQtZjE5ODI2OTJhNTU0XkEyXkFqcGdeQXVyMTQxNzMzNDI@._V1_QL75_UY562_CR2,0,380,562_.jpg"
          />
          <img
            alt="Movie 21"
            className="image-item"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BMDliMmNhNDEtODUyOS00MjNlLTgxODEtN2U3NzIxMGVkZTA1L2ltYWdlXkEyXkFqcGdeQXVyNjU0OTQ0OTY@._V1_QL75_UX380_CR0,0,380,562_.jpg"
          />
          <img
            alt="Movie 22"
            className="image-item"
            onClick={[Function]}
            src="https://m.media-amazon.com/images/M/MV5BOTA5NDZlZGUtMjAxOS00YTRkLTkwYmMtYWQ0NWEwZDZiNjEzXkEyXkFqcGdeQXVyMTMxODk2OTU@._V1_QL75_UX380_CR0,0,380,562_.jpg"
          />
        </div>
        <button
          className="slide-button material-symbols-rounded"
          id="next-slide"
        >
          chevron_right
        </button>
      </div>
      <div
        className="slider-scrollbar"
      >
        <div
          className="scrollbar-track"
        >
          <div
            className="scrollbar-thumb"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
